pages/index.js

```diff
@@ -39,11 +39,16 @@ const StyledH1 = styled.h1`
 `;
 
 const StyledList = styled.ul`
-  list-style-type: none;
+  display: flex;
+  flex-direction: column;
   padding: 0;
+  gap: 12px;
+  align-content: start;
+  list-style-type: none;
 `;
 
 const CategoryBox = styled.span`
+  display: flex;
   position: absolute;
   top: 5px;
   right: 5px;
```

Comment:
```suggestion
```
Just a tipp (If you want that): If you remove those 3 -> your category box gets aligned in the middle because of the flex container.